vizro-core/examples/demo/app.py

```diff
@@ -438,7 +438,7 @@ def create_benchmark_analysis():
         components=[
             vm.AgGrid(
                 title="Click on a cell in country column:",
-                figure=dash_ag_grid(id="dash_ag_grid_country", data_frame=gapminder, columnDefs=columnsDefs),
+                figure=dash_ag_grid(data_frame=gapminder, columnDefs=columnsDefs),
```

Comment:
Could you bring these changes over to vizro-internal as well? 🙏 To keep the deployed version in sync

Reply:
Will do, should I deploy it, or di we wait until a release?